### ISSUE TITLE
osa_differ commit 77014f847c8cff557fc5e38e99880844de42ad84 breaks rpc_differ

### ISSUE THREAD

**Issue by alextricity25** [rcbops/rpc_differ#3]
osa_differ commit https://github.com/major/osa_differ/commit/77014f847c8cff557fc5e38e99880844de42ad84 removes a  parameter from the ``get_projects`` method, but the method call here was never updated to reflect those changes:
https://github.com/major/rpc_differ/blob/master/rpc_differ/rpc_differ.py#L275
This will cause the script to fail with the following message for anyone running the latest version of rpc_differ with the latest version of osa_differ(which is what will happen when you download rpc_differ from pip with the current requirements constriant).
```
Traceback (most recent call last):
  File "/home/alex/Dropbox/RackSpace/Devlopment/rpc_differ_devel/venvs/bin/rpc-differ", line 11, in <module>
    load_entry_point('rpc-differ', 'console_scripts', 'rpc-differ')()
  File "/home/alex/Dropbox/RackSpace/Devlopment/rpc_differ_devel/rpc_differ/rpc_differ/rpc_differ.py", line 294, in run_rpc_differ
    osa_old_commit)
TypeError: get_projects() takes exactly 2 arguments (3 given)
```

**Comment by alextricity25** [comment]
PR is #4

**Comment by alextricity25** [comment]
This was fixed in #4
